### DISCUSSION TITLE
time zone?

### DISCUSSION THREAD

**Discussion by nqnga** [highfestiva/finplot#309] [Q&A] [answered]
HI, Can I ask what is the default time zone in your library? my original data timezone is +5:30. 
![image](https://user-images.githubusercontent.com/9681519/158796027-e2735f1a-cfea-4e4a-9720-2550ad78f522.png)
I converted the to london time 
![image](https://user-images.githubusercontent.com/9681519/158796370-aacf090d-4c5e-4323-9189-5048c03c624b.png)

in the chart showed different time than in the data?
![image](https://user-images.githubusercontent.com/9681519/158796153-825fc4d4-1e1f-4b54-bf3e-afa7abcfb00a.png)
Please advise me how to dix this issue?
Thanks

**Comment by highfestiva** [comment, marked as answer]
Does [this](https://github.com/highfestiva/finplot/wiki/Snippets#time-zone) help?

**Reply by nqnga** [reply]
HI, I tried many times, but it does not change: sometime it works sometime it does not :D
![image](https://user-images.githubusercontent.com/9681519/163026958-f2768390-9d95-4229-8042-fe0cf4ac454f.png)
 and on double click:
![image](https://user-images.githubusercontent.com/9681519/163027051-d7dd0a3c-f88c-4297-8f62-3cb517a32a80.png)
 totally different time:(

**Comment by highfestiva** [comment]
Default display is local timezone. So if you have data in local timezone, it will be displayed that way too. If you want to change to UTC and display in UTC you know how to do that to. I distrust the statement that is varies. I TZ fix is upcoming though, but that only concerns DST and crossing DST boundaries.

**Reply by nqnga** [reply]
I changed timezone in my laptop to newyork location, it is still the same. I am waiting for the update then.
Can I ask how can I plot multiple charts also seting onclick to get data from each chart?
![image](https://user-images.githubusercontent.com/9681519/163408910-8d340401-2f7d-43a7-90ca-d1c19f4e67b0.png)
like this, after i drew, i want to click to get data on each chart I used fplt.set_time_inspector(get_info, when='dclick') this function, but i can only get 1 chart, what should i do?
Thanks

**Reply by highfestiva** [reply]
Supply `ax=ax1`, `ax=ax2` and so forth to `set_time_inspector()` if you want to have a click feedback for each. Print any data you like when clicked.

**Reply by armeya25** [reply]
dataframe['date'] = pd.to_datetime(dataframe['date'])
dataframe['date'] = dataframe['date'].dt.tz_localize(None)
dataframe.set_index('date', inplace=True)

this helped me
searched for two days for it almost lost my head

i have something like this ->

date,open,high,low,close,volume
2015-02-02 09:15:00+05:30,361.05,363.95,357.3,358.1,39042
2015-02-02 09:25:00+05:30,358.1,358.8,356.5,357.05,33574
2015-02-02 09:35:00+05:30,357.05,359.95,356.55,359.8,21728
2015-02-02 09:45:00+05:30,359.8,359.9,358.1,359.8,15207
2015-02-02 09:55:00+05:30,359.65,361.6,359.25,359.7,84030
2015-02-02 10:05:00+05:30,359.7,360.6,358.2,358.45,82249
